tools/article_checker/claude_retriever/utils.py

```
@@ -43,8 +44,22 @@ def format_results_full(extracted: list[str]) -> str:
+                     missing_content_placeholder: str = "CONTENT NOT AVAILABLE",
+                     max_retries: int = 2) -> str:
+    """
+    Scrape a URL with retry logic and optional Claude summarization.
+    """
+    content = None
+    for attempt in range(max_retries + 1):
+        try:
+            content = await get_url_content(url)
+            if content:
+                break
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)